sdk/nodejs/output.ts

```diff
@@ -775,9 +775,6 @@ export const Output: OutputConstructor = <any>OutputImpl;
  * ```
  */
 export type Lifted<T> =
-    // Output<T> is an intersection type with 'Lifted<T>'.  So, when we don't want to add any
-    // members to Output<T>, we just return `{}` which will leave it untouched.
-    T extends Resource ? {} :
```

Comment:
remove special hardcoding where we basically suppressed lifting on Resources (from teh type system perspective).